azure/Guidelines.md

```diff
@@ -760,79 +760,64 @@ while the Microsoft guidelines use the name "Operation-Location".
 
 :white_check_mark: **DO** look for **both** HEADERS in client code, preferring the `Operation-Location` version.
 
-### Bring your own Storage
-When implementing your service, it is very common to store and retrieve data and files. When you encounter this scenario, avoid implementing your own storage strategy and instead use Azure Bring Your Own Storage (BYOS). BYOS provides significant benefits to service implementors, e.g. security, an aggressively optimized frontend, uptime, etc.
-While Azure Managed Storage may be easier to get started with, as your service evolves and matures, BYOS will provide the most flexibility and implementation choices. Further, when designing your APIs, be cognizant of expressing storage concepts and how clients will access your data. For example, if you are working with blobs, then you should not expose the concept of folders, nor do they have extensions.
+### Bring your own Storage (BYOS)
+Many services need to store and retrieve data files. For this scenario, teh service should not implement its own 
```

Comment:
My strong opinion is that BYOS should be an option not a requirement. Forcing the user to create and provide a storage account with the required permissions etc. drastically raises the bar for starting to use a simple service. This is also what service teams find when they talk to customers. Both service managed storage and BYOS are valid scenarios.

Reply:
I read this as warning against adding your own API façade on top of Blobs like we see so many teams do.  If they want to go through the work for Managed Storage, that's great and will improve the getting started experience for their customers.  But if they try to add their own `UploadFile` API for files larger than around 256MB, they're probably going to do a much worse job than the Storage team's end to end developer experience and should just accept a SAS URI to a blob.

Reply:
Yes, I was trying to say what Ted is saying (don't write your own API on top of Azure storage).
I'm OK if teams want to use managed storage too. We can add a sentence to this effect in the intro but then the team is writing a storage API. We can discuss but I'm OK with all of this.